schema/filter_artworks.js

```diff
@@ -186,6 +216,7 @@ function filterArtworks(primaryKey) {
       }
 
       return gravity.with(accessToken)("filter/artworks", gravityOptions)
+        .then(response => assign({}, response, { options: gravityOptions }))
```

Comment:
Then, rendering some sort of title like 'Large For-Sale Animal Works under $1000' is done just from the current filter state, vs parsing these returned parameters. Is it just something about the Relay spec/query formulation?